.github/workflows/ci.yml

```diff
@@ -221,27 +221,37 @@ jobs:
     name: cross
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
         target:
+          - aarch64-unknown-linux-gnu
+          - armv5te-unknown-linux-gnueabi
           - i686-unknown-linux-gnu
-          - powerpc-unknown-linux-gnu
-          - powerpc64-unknown-linux-gnu
-          - mips-unknown-linux-gnu
-          - arm-linux-androideabi
+          - mips64el-unknown-linux-gnuabi64
+          - mipsel-unknown-linux-gnu
+          - powerpc64le-unknown-linux-gnu
+          - riscv64gc-unknown-linux-gnu
+          - s390x-unknown-linux-gnu
```

Comment:
The list of targets are based on architectures officially supported by debian,ubuntu,fedora

https://wiki.debian.org/SupportedArchitectures (the following list doesn't include unofficially supported arch)
- x86
- x86_64
- arm
- aarch64
- mips64el
- mips ([deprecated in debian 10](https://wiki.debian.org/MIPSPort#Support))
- mipsel
- powerpc64le
- s390x

https://help.ubuntu.com/community/SupportedArchitectures
- x86
- x86_64
- arm
- aarch64
- powerpc64le
- s390x

https://fedoraproject.org/wiki/Architectures
- x86_64 (Primary Architecture)
- arm (Primary Architecture)
- aarch64 (Primary Architecture)
- mips64el 
- mipsel
- powerpc64le
- riscv64gc
- s390x